ui/src/app/store/application/reducers.ts

```diff
@@ -49,15 +51,24 @@ export const initialState: AppEntityState = applicationAdapter.getInitialState({
 
 const reducer: ActionReducer<AppEntityState> = createReducer(
   initialState,
-  on(loadApplications, createApplication, updateApplication, deleteApplication, state => ({ ...state, isPending: true })),
+  on(loadApplications, createApplication, updateApplication, deleteApplication, refreshApplication, state => ({
+    ...state,
+    isPending: true,
+  })),
   on(loadApplicationsFail, createApplicationFail, updateApplicationFail, deleteApplicationFail, state => ({ ...state, isPending: false })),
   on(createApplicationSuccess, (state, { application }) => applicationAdapter.addOne(application, { ...state, isPending: false })),
   on(loadApplicationsSuccess, (state, { applications }) => applicationAdapter.setAll(applications, { ...state, isPending: false })),
   on(updateApplicationSuccess, (state, { application }) =>
     applicationAdapter.updateOne({ id: application.id, changes: application }, { ...state, isPending: false })
   ),
   on(deleteApplicationSuccess, (state, { id }) => applicationAdapter.removeOne(id, { ...state, isPending: false })),
-  on(setSelectedAppIdEntityAction, (state, { selectedAppId }) => ({ ...state, selectedAppId }))
+  on(setSelectedAppIdEntityAction, (state, { selectedAppId }) => ({ ...state, selectedAppId })),
+  on(refreshApplication, (state, { userId, lastName, firstName }) => {
```

Comment:
there is already refreshApplication handler above which will set isPending to true, and this one immediately will change it to false

Reply:
Could you please clarify, what are you mean?

Reply:
there are 2 refreshApplication handlers in this reducer

Reply:
Ok thanks, fixed